.gitignore

```diff
@@ -84,9 +85,6 @@ jspm_packages
 # Webstorm
 .idea
 
-# VS Code
-.vscode

```

Comment:
Any reason you've removed the .vscode directory from the ignore?

Reply:
I just copied a .gitignore file from another plugin project.

Reply:
The old .gitignore file wasn't ignoring /package-lock.json (which is what I was looking for).

Reply:
I've restored the .vscode exclusion.